apps/dotcdn/src/app/app.module.ts

```diff
@@ -0,0 +1,70 @@
+import { NgModule } from '@angular/core';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { BrowserModule } from '@angular/platform-browser';
+import { InputTextModule } from 'primeng/inputtext';
+import { DropdownModule } from 'primeng/dropdown';
+import { AppComponent } from './app.component';
+import {
+    CoreWebService,
+    DotcmsConfigService,
+    DotcmsEventsService,
+    DotEventsSocket,
+    DotEventsSocketURL,
+    LoggerService,
+    LoginService,
+    SiteService,
+    StringUtils
+} from '@dotcms/dotcms-js';
+import { ReactiveFormsModule } from '@angular/forms';
+import { RouterModule } from '@angular/router';
+import { HttpClientModule } from '@angular/common/http';
+import { SkeletonModule } from 'primeng/skeleton';
+import { TabViewModule } from 'primeng/tabview';
+import { ChartModule } from 'primeng/chart';
+import { InputTextareaModule } from 'primeng/inputtextarea';
+import { ButtonModule } from 'primeng/button';
+import { FormsModule } from '@angular/forms';
+import { DotSpinnerModule, DotIconModule } from '@dotcms/ui';
+import { DotCDNStore } from './dotcdn.component.store';
+
+const dotEventSocketURLFactory = () => {
```

Comment:
this here ?

Reply:
Yeah, sadly we had to do it that way. I'm open to suggestion, tho.